openreview/venue_request/process/revisionProcess.py

```diff
@@ -194,6 +194,20 @@ def process(client, note, invitation):
             ))
 
         elif invitation_type == 'Submission_Revision_Stage':
+            submission_revision_stage_notes = client.get_references(
+                referent=forum_note.id,
+                invitation='{support_group}/-/Request{number}/Submission_Revision_Stage'.format(
+                    support_group=SUPPORT_GROUP, number=forum_note.number),
+                limit=2
+            )
+            if len(submission_revision_stage_notes) > 1:
+                last_submission_revision_stage_note = submission_revision_stage_notes[-1]
+                expire_revision_stage_name = last_submission_revision_stage_note.content.get('submission_revision_name',
+                                                                                             'Revision')
+            else:
+                expire_revision_stage_name = 'Revision'
+            if expire_revision_stage_name != forum_note.content.get('submission_revision_name', '').strip():
+                conference.expire_invitation(conference.get_invitation_id(expire_revision_stage_name))
```

Comment:
When the old revision name is something like `Camera Ready Revision`, you need to replace spaces with `_`, otherwise you won't find the invitation and they won't be expired:

https://github.com/openreview/openreview-py/blob/71f501c59103089a6e18fad27da5e8b1f2f2d497/openreview/conference/helpers.py#L419